<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="CopiousCare" failures="1" tests="16" name="webAutoNegative.EditAndCancel" time="146.427" errors="0" timestamp="2024-04-03T12:02:05 IST" skipped="0">
  <testcase classname="webAutoNegative.EditAndCancel" name="numberField" time="0.110"/>
  <system-out/>
  <testcase classname="webAutoNegative.EditAndCancel" name="sendOTP" time="8.505"/>
  <system-out/>
  <testcase classname="webAutoNegative.EditAndCancel" name="verifyButton" time="10.105"/>
  <system-out/>
  <testcase classname="webAutoNegative.EditAndCancel" name="searchBar" time="0.379"/>
  <system-out/>
  <testcase classname="webAutoNegative.EditAndCancel" name="searchOpt" time="3.057"/>
  <system-out/>
  <testcase classname="webAutoNegative.EditAndCancel" name="clickPatient" time="10.263"/>
  <system-out/>
  <testcase classname="webAutoNegative.EditAndCancel" name="firstName" time="1.226"/>
  <system-out/>
  <testcase classname="webAutoNegative.EditAndCancel" name="nextButton" time="33.944"/>
  <system-out/>
  <testcase classname="webAutoNegative.EditAndCancel" name="BirthHistory" time="1.915"/>
  <system-out/>
  <testcase classname="webAutoNegative.EditAndCancel" name="BirthHistory1" time="1.740"/>
  <system-out/>
  <testcase classname="webAutoNegative.EditAndCancel" name="BirthHistoryCancel" time="15.435"/>
  <system-out/>
  <testcase classname="webAutoNegative.EditAndCancel" name="HealthProfile" time="26.790"/>
  <system-out/>
  <testcase classname="webAutoNegative.EditAndCancel" name="HealthProfileCancel" time="2.277"/>
  <system-out/>
  <testcase classname="webAutoNegative.EditAndCancel" name="familyProfile" time="18.057"/>
  <system-out/>
  <testcase classname="webAutoNegative.EditAndCancel" name="familyProfileCancel" time="12.124"/>
  <system-out/>
  <testcase classname="webAutoNegative.EditAndCancel" name="ProfileValidation" time="0.500">
    <failure message="expected [Automation] but found []" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Automation] but found []
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at webAutoNegative.EditAndCancel.ProfileValidation(EditAndCancel.java:672)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- ProfileValidation -->
  <system-out/>
</testsuite> <!-- webAutoNegative.EditAndCancel -->
